### PR TITLE
0x6j/device binding iOS

### PR DESCRIPTION
**Pull request by commjoen**
This is a first pass for review for the device binding

### CODE REVIEW

**Comment by commjoen** [comment]
This is a first PR for https://github.com/OWASP/owasp-mstg/issues/396 it can be used to give some comment so I can finish this off today.

File: Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 1]
```diff
@@ -1153,6 +1153,51 @@ Refer to the "Tampering and Reverse Engineering section" for examples of patchin
 
 The goal of device binding is to impede an attacker when he tries to copy an app and its state from device A to device B and continue the execution of the app on device B. When device A has been deemend trusted, it might have more privileges than device B, which should not change when an app is copied from device A to device B.
 
+#### Static Analysis
+
+In the past, Android developers often relied on the Secure ANDROID_ID (SSAID) and MAC addresses. However, the behavior of the SSAID has changed since Android O and the behavior of MAC addresses have changed in Android N <sup>[1]</sup>. Google has set a new set of recommendations in their SDK documentation regarding identifiers as well <sup>[2]</sup>.
```
Comment (bolot) [inline Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: This paragraph appears to be duplicated below.

**Comment by sushi2k** [comment]
Great content @commjoen 
Just had view formatting issues and maybe you can outline the commands in code tags needed to e.g. copy data from the devices

**Comment by commjoen** [comment]
@sushi2k  & @bolot  should I fix it in this branch ? (Which has been merged) or should i create a new one?

**Pull request merged**